schema/spacewalk/upgrade/susemanager-schema-5.1.7-to-susemanager-schema-5.1.8/300-fix-config-channel-position.sql

```diff
@@ -3,22 +3,22 @@ UPDATE suseStateRevisionConfigChannel
 SET position = (
     SELECT rank
     FROM (
-        SELECT state_revision_id, config_channel_id, position, (row_number() OVER (PARTITION BY state_revision_id ORDER BY position ASC)) rank
+        SELECT state_revision_id, config_channel_id, position, (row_number() OVER (PARTITION BY state_revision_id ORDER BY position ASC NULLS LAST)) rank
         FROM suseStateRevisionConfigChannel scc
```

Comment:
Should the position not null be filtered also in this sub-query, to make it more robust?

Reply:
NULLS LAST has the same effect, I think.